source/deployment-paradigms/multi-region.txt

```diff
@@ -0,0 +1,252 @@
+down, the traffic can be served from the same region to ensure the same latency 
+on the primary region. 
+
+.. figure:: /includes/images/multi-region-5+3.png
+   :figwidth: 750px
+   :alt: An image showing three types of multi-region deployments
+
+Notes and Considerations
+````````````````````````
+
+- Deploy your app servers in each region that MongoDB nodes are 
+  deployed. This gives you the ability to route read requests to the in-region  
+  node, which reduces the response time to your end-users, and offloads requests 
+  from the primary node. Write requests are always be directed to the primary.
+
+- Use :atlas:`private endpoints <data-federation/admin/manage-private-endpoint/>`
+  for connecting to the cluster, and 
+  :atlas:`VPC peering <atlas-stream-processing/manage-vpc-peering-connections/>` 
+  between your app server VPCs. VPC peering ensures that if a network connection 
+  is broken or {+service+} in that region goes down, the application tier can 
+  still route to the primary node, first over the VPC peering, and then over the 
+  private endpoint. 
+
+- This architecture has the highest cost due to network traffic between regions 
+  and having 5 or more data-bearing nodes. 
+
+- This architecture provides the highest resiliency. There are no interruptions 
+  during {+service+} operations (like an automated upgrade), and your application 
+  can sustain a full regional failure with no interruption and manual 
+  intervention required. 
+
+.. _arch-center-multi-region-5+2:
+
+5-Node, 2-Region Architecture
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have an application that can experience limited downtime or maintenance 
+windows without significant revenue impact, you can create an architecture that 
+provides that level of availability by using 5 nodes in 2 regions. You have 3 
+electable nodes spread across the 2 regions, and also have 2 read-only nodes in 
+the secondary region. The electable nodes are in the **majority region**, while 
+the region with fewer electable nodes is the **minority region**.
+
+This is also an appropriate architecture if you need high availability and low 
+latency in a geography that only has 2 regions.
+
+This is a good option for:
+
+- Customers with only 2 approved regions.
+- Meeting governance compliance about data storage.
+- Applications where read operations need high uptime, but write operations can 
+  experience some outage.
+
+.. figure:: /includes/images/multi-region-5+2.png
+   :figwidth: 750px
+   :alt: An image showing three types of multi-region deployments
+
+Notes and Considerations
+````````````````````````
+This architecture provides increased protection 
+against data loss, even in full regional outage. The system remains available 
+in read-only mode in the secondary region if the primary region is lost. 
+
+While this architecture is less expensive than the 5-Node, 3-Region solution 
+provided above, the reduced cost comes with some caveats:
```

Comment:
Should these be bullet points?

Reply:
I'm not sure I agree here. I've changed the first para to make it clear that the second sentence is a further description of the first sentence, not a separate point to be made. Thoughts?